client/app/services/custom-button/custom-button.html

```diff
@@ -4,6 +4,7 @@
           ng-click="vm.invokeCustomAction(button)"
           uib-tooltip="{{button.description}}"
           tooltip-placement="top">
+    <i ng-class="button.options.button_icon" ng-style="{color: button.options.button_color }"></i>&nbsp;
```

Comment:
So has the API been updated to send the icons in the custom button API response?